packages/perseus-editor/src/editor.tsx

```diff
@@ -211,20 +210,11 @@ class Editor extends React.Component<Props, State> {
         // will return false. However at least in Firefox setting `value` on a
         // textbox clears the undo stack so we don't get unexpected undo
         // behavior.
-        if (this.lastUserValue !== null && textarea) {
-            /**
-             * TODO(somewhatabstract, JIRA-XXXX):
-             * textarea should be refined with an instanceof check to
-             * HTMLTextAreaElement so that these props are available.
-             */
-            // @ts-expect-error - TS2339 - Property 'focus' does not exist on type 'Element | Text'.
+        if (this.lastUserValue != null && textarea) {
             textarea.focus();
-            // @ts-expect-error - TS2339 - Property 'value' does not exist on type 'Element | Text'.
             textarea.value = this.lastUserValue;
-            // @ts-expect-error - TS2339 - Property 'selectionStart' does not exist on type 'Element | Text'.
             textarea.selectionStart = 0;
-            // @ts-expect-error - TS2339 - Property 'select' does not exist on type 'Element | Text'.
-            textarea.select(0, prevProps.content.length);
```

Comment:
`.select()` ([docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/select)) on the `<textarea>` element doesn't take any arguments, but `setSelectionRange` ([docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/setSelectionRange)) does!